<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="Level">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>62</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lifeValue">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>95</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="attackValue">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>128</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="defenceValue">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>161</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="money">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>194</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="goldKey">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>260</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="experience">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>227</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="blueKey">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>293</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="floor">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>359</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="redKey">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>326</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
